### PR TITLE
Add mutation from `!!foo` to `foo`

### PR DESCRIPTION
**Pull request by backus**

### CODE REVIEW

**Comment by backus** [comment]
@mbj this is ready for review

File: lib/mutant/mutator/node/send.rb [hunk 1]
```diff
@@ -126,6 +127,18 @@ def emit_receiver_selector_mutations
...
+        # Emit mutation from `!!foo` to `foo`
+        #
+        # @return [undefined]
+        def emit_double_negation_mutation
+          return unless selector.equal?(:!) && n_send?(receiver)
+
+          negated = AST::Meta::Send.new(meta.receiver)
+          return unless negated.selector.equal?(:!)
+
+          emit(negated.receiver)
```
Comment (mbj) [inline lib/mutant/mutator/node/send.rb]: Should this implicit return the explicitly guarded to remove the eager return?

Comment (backus) [inline lib/mutant/mutator/node/send.rb]: Yup

**Comment by mbj** [comment]
@backus We forgot the changelog. Feel free to PR and merge that one yourself.

**Comment by backus** [comment]
@mbj alright. I should probably do a pass over changes since last release in general. I think many things are not in the changelog

**Comment by mbj** [comment]
@backus Unfortunate but correct.

**Pull request merged**